src/components/app/App.js

```diff
@@ -2,21 +2,28 @@ import React from 'react';
 import { Header } from '../header/HeaderFromLecture';
 
 // todo: здесь нужно сделать импорт Footer из '../footer/Footer'
+import {Footer} from "../footer/Footer";
 
 // todo: здесь нужно сделать импорт usersList из '../../constants'
+import {usersList} from "../../constants";
 // todo: здесь нужно сделать импорт UserCard из '../user-card/UserCard'
 
+
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import {UserCard} from "../user-card/UserCard";
```

Comment:
This import should occur before imports of styles. So please move this import to line 10